manifests/params.pp

```diff
@@ -25,6 +25,7 @@
   $conf_dir_purge                 = false
   $apt_default_release            = 'stable'
   $apm_default_enabled            = false
+  $apm_analyzed_spans_list        = []
```

Comment:
We might or might not need this, depending on if you decide to implement `$apm_analyzed_spans` above as an `Optional`.

Reply:
This is now not listed in the params.pp file.